gatsby/content/blog/2023/03/2023-03-31-twim.mdx

```diff
@@ -0,0 +1,281 @@
+> 
+> For an idea of what's ahead, here's what the SCT will be looking at over the next several months:
+> 
+> * Linearized Matrix (implementation & MSC)
+> * Extensible Events (at least the core types) - this will serve as the basis for an interoperable messaging format in our IETF drafts
+> * Decentralized MLS & interoperability of crypto
+> * Clarification gaps and bugs in the current spec
+> * Pseudonymous user IDs and account portability
+> * Almost certainly something that was missed when writing this list
+> 
+> Considering the above list, the Matrix 2.0 objectives (sliding sync, OIDC, native VoIP conferencing, and faster room joins), and the core team's work around mentions, abuse reporting, and more, the SCT will be a bit busy. That said, if you have MSCs you think we should be looking at, let us know in the [#sct-office:matrix.org](https://matrix.to/#/#sct-office:matrix.org) room. We've recently started doing our weekly planning in that room too, which should help give an idea for what the SCT is expecting to look at each week.
+> 
+> ## Random MSC of the Week
+> 
+> The random MSC of the week is... [\[WIP\] MSC2966: Usage of OAuth 2.0 Dynamic Client Registration in Matrix](https://github.com/matrix-org/matrix-spec-proposals/pull/2966)!
+> 
+> This MSC provides a mechanism for implementing Dynamic Client Registration ([RFC 7591](https://tools.ietf.org/html/rfc7591)) for OAuth 2.0 between Matrix clients and homeservers. Without this, homeserver admins would need to manually configure OAuth metadata (Redirect URIs, application names, client secrets, [and more](https://www.iana.org/assignments/oauth-parameters/oauth-parameters.xhtml#client-metadata)) for every Matrix client that wanted to connect to the homeserver. Since that doesn't really scale in an environment that allows anyone to use any client they like, dynamic registration is critical! Dynamic registration allows clients to communicate this metadata to the homeserver at the login/registration step.
+> 
+> [MSC2966](https://github.com/matrix-org/matrix-doc/issues/2966) is part of a series of MSCs that add first-class OpenID Connect (OIDC) support to Matrix. You can see an overview of the related MSCs ([here](https://github.com/matrix-org/matrix-spec-proposals/pull/3861)) and https://areweoidcyet.com/ for the latest progress on integrating OIDC into the Matrix spec!
+
+## Dept of Servers 🏢
+
+### Synapse ([website](https://github.com/matrix-org/synapse/))
+
+Synapse is a Matrix homeserver implementation developed by the matrix.org core team
+
+[Nico](https://matrix.to/#/@deepbluev7:neko.dev) says
```

Comment:
Shouldnt this be below the synapse update infos? Seems weird to have the story first :)

Reply:
Why not :)